spec/rubocop/cli/disable_uncorrectable_spec.rb

```diff
@@ -282,6 +282,32 @@ def our_function
             # rubocop:enable Layout/LineLength
           RUBY
         end
+
+        it 'adds before-and-after disable statement around the multi-line percent array' do
+          create_file('.rubocop.yml', <<~YAML)
+            Layout/LineLength:
+              Max: 30
+          YAML
```

Comment:
Can you move this `create_file('.rubocop.yml', <<~YAML)` under L263 and put it in `before` block?

```diff
diff --git a/spec/rubocop/cli/disable_uncorrectable_spec.rb b/spec/rubocop/cli/disable_uncorrectable_spec.rb
index b1d42a469..e2003225b 100644
--- a/spec/rubocop/cli/disable_uncorrectable_spec.rb
+++ b/spec/rubocop/cli/disable_uncorrectable_spec.rb
@@ -261,11 +261,14 @@ RSpec.describe 'RuboCop::CLI --disable-uncorrectable', :isolated_environment do
       end

       context 'and the offense is inside a percent array' do
-        it 'adds before-and-after disable statement around the percent array' do
+        before do
           create_file('.rubocop.yml', <<~YAML)
             Layout/LineLength:
               Max: 30
           YAML
+        end
+
+        it 'adds before-and-after disable statement around the percent array' do
           create_file('example.rb', <<~RUBY)
             # frozen_string_literal: true

@@ -284,10 +287,6 @@ RSpec.describe 'RuboCop::CLI --disable-uncorrectable', :isolated_environment do
         end

         it 'adds before-and-after disable statement around the multi-line percent array' do
-          create_file('.rubocop.yml', <<~YAML)
-            Layout/LineLength:
-              Max: 30
-          YAML
           create_file('example.rb', <<~RUBY)
             # frozen_string_literal: true
```